code/modules/mob/delays.dm

```diff
@@ -5,7 +5,7 @@
 // Reduces duplicated code by quite a bit.
 /datum/delay_controller
 	// Delay clamps (for adminbus, effects)
-	var/min_delay = 1
+	var/min_delay = 0.001
```

Comment:
Well I've tried, and no matter how fast I set the modifier to be, I don't get any teleporting.